### PR TITLE
set runtime:server as default when booting app

### PR DESCRIPTION
**Pull request by lzhan**
functional test is added.

### CODE REVIEW

File: tests/func/applications/frameworkapp/configapp/routes.json [hunk 1]
```diff
@@ -11,5 +11,13 @@
         "path": "/:mojit-id/:mojit-action",
         "call": "{mojit-id}.{mojit-action}"
     }
+  },
```
Comment (caridy) [inline tests/func/applications/frameworkapp/configapp/routes.json]: we are removing routes.json contextualization, better to use a different file for the test.

File: lib/store.js [hunk 2]
```diff
@@ -56,6 +56,7 @@ Store.createStore = function(options) {
     if (!options.preload) {
         options.preload = 'full';
     }
+    options.context = {runtime: 'server'};
```
Comment (caridy) [inline lib/store.js]: context might be set, instead, you can do this:

```
if (!options.context.runtime) {
    options.context.runtime = 'server';
}
```

that will work in most cases, even when you want to force runtime to be another thing from the cli, etc.

**Comment by lzhan** [comment]
travis failed, but it is not related to the pr.

**Comment by caridy** [comment]
+1 please, verify that travis passes before merging.

TODO
- [x] History entry is still missing

**Comment by lzhan** [comment]
travis is fixed.

**Pull request merged**